introduction.md

```diff
@@ -0,0 +1 @@
+# Hi my name is Jeanie and I am eager to learn git
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.